backend/oasst_backend/utils/hugging_face.py

```diff
@@ -63,3 +69,24 @@ async def post(self, input: str) -> Any:
                 inference = await response.json()
 
         return inference
+
+
+async def save_toxicity(
+    interaction,
+    pr: PromptRepository,
+    new_message: Message,
+):
+    try:
+        model_name = HfModel.TOXIC_ROBERTA.value
+        hugging_face_api = HuggingFaceAPI(f"{HfUrl.HUGGINGFACE_TOXIC_ROBERTA.value}/{model_name}")
```

Comment:
Again, I don't think that results in the correct endpoint (at least I get a "not found" when I go there)

Reply:
Solved thanks!